klippy/extras/load_cell_probe.py

```diff
@@ -0,0 +1,611 @@
+# Support for bed level probes featuring a load cell at which the hotend is
+# suspended.
+#
+# Copyright (C) 2022 Martin Hierholzer <martin@hierholzer.info>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+import mathutil
+import sys
+import logging
+import random
+import math
+
+syspath = sys.path
+sys.path = syspath[1:]
+import statistics
+sys.path = syspath
```

Comment:
It's not clear to me what this does.  It does not look like something we want to have in the code.